templates/tools/dockerfile/test/python_stretch_default_x64/Dockerfile.template

```diff
@@ -0,0 +1,36 @@
+%YAML 1.2
+--- |
+  # Copyright 2018 The gRPC Authors
+  #
+  # Licensed under the Apache License, Version 2.0 (the "License");
+  # you may not use this file except in compliance with the License.
+  # You may obtain a copy of the License at
+  #
+  #     http://www.apache.org/licenses/LICENSE-2.0
+  #
+  # Unless required by applicable law or agreed to in writing, software
+  # distributed under the License is distributed on an "AS IS" BASIS,
+  # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  # See the License for the specific language governing permissions and
+  # limitations under the License.
+
+  <%include file="../../python_stretch.include"/>
+
+  RUN apt-get update && apt-get install -y zlib1g-dev libssl-dev
+  RUN apt-get update && apt-get install -y jq build-essential libffi-dev
+
+  RUN cd /tmp && ${'\\'}
+    wget -q https://github.com/python/cpython/archive/v3.6.9.tar.gz && ${'\\'}
```

Comment:
Optional: Can we do a checksum verification on the tarball to guarantee hermeticity across changes to the `Dockerfile`?